.travis.yml

```diff
@@ -14,7 +14,7 @@ cache:
 
 matrix:
     fast_finish: true
-    allow_failures:
+    include:
```

Comment:
This PR should be opened against 1.9 as you allowed failures there in the first place.